cloudman/projman/management/commands/projman_create_project.py

```diff
@@ -25,6 +26,7 @@ def create_project(name):
             if not pmapi.projects.find(name):
                 pmapi.projects.create(name)
                 print("Project created successfully.")
+            call_command("create_namespace", name)
```

Comment:
I think it would be better to create the namespace as part of the process of creating the project. We should also probably disallow creating projects with names that match namespaces that already exist, to prevent users from creating a project like "cloudman" and thus being able to access the charts within.